package/MDAnalysis/core/groups.py

```diff
@@ -816,6 +816,24 @@ def wrap(self, compound="atoms", center="com", box=None):
             if not all(s == 0.0):
                 o.atoms.translate(s)
 
+    def groupby(self, topattr):
+        """Obtain groupings of the components of this Group according the values
```

Comment:
This should be a one liner, so something like 'group according to a given attribute'